Procfile

```diff
@@ -1 +1,2 @@
-web: node lib/main production
+web:                     node lib/main server
+expireCacheBusters: node lib/main expire-cache-busters
```

Comment:
hmm... You might be right.. as it's all cached it doesn't really matter that we use multiple entities...